src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,7 @@ Pending
 
 0.5.146
 +++++++
+* Extend containerinsights --data-collection-settings with new fields "streams" and containerlogv2
```

Comment:
Please choose a new version number (and update setup.py) or move this to the pending section.